src/main/java/employee/utility/EmployeeUtil.java

```diff
@@ -0,0 +1,49 @@
+package employee.utility;
+
+public class EmployeeUtil {
+
+    public int checkHeader(String contentType) { // application/vnd.pl.employee+json; version=1.0
+
+        String[] contents = contentType.split("/|;"); // 0. application (ignore) 1. vendor+returnTYpe & 2. version
+        System.out.println("contents.length: " + contents.length);
+        int returnVal;
+
+        String[] vendorType = contents[1].split("\\+"); // 0. vendor 1. returnType
+        if(vendorType.length > 1) {
+
+            if(vendorType[0].equals("vnd.pl.employee")) {
+
+                if(vendorType[1].equals("json")) {
+
+                    if(contents.length >= 3) {
+                        String[] version = contents[2].split("=");
+                        if(version.length > 1) {
+                            returnVal = (int) Double.parseDouble(version[1]);
```

Comment:
So this is interesting, the string version is `1.0` which is a double, there might be a 1.2 version which casting to an `int` will get rid of it. You might as well keep the version as a double the whole way through, no harm in it.

Reply:
What if we keep it as a `String`? if we have a version number 1.0.2 for example

Reply:
Good question, you can enumerate strings, though numbers have a kind of ordering to it. Both work in this case, but usually these version numbers tend to stay very very simple. it's almost always 1 or 2.